lib/potassium/assets/testing/devise_config.rb

```diff
@@ -0,0 +1,5 @@
+require 'devise'
+
+RSpec.configure do |config|
+  config.include Devise::Test::ControllerHelpers, type: :controller
```

Comment:
I remember in a project that I wanted to use `sign_in` in a swagger request test, I needed to add [IntegrationHelpers](https://www.rubydoc.info/gems/devise/Devise/Test/IntegrationHelpers). I added this to the `swagger_helper.rb`:

```
config.include Devise::Test::IntegrationHelpers, type: :request
```

What do you think, should we add something like that?